content/06.prior-knowledge.md

```diff
@@ -1,27 +1,26 @@
 ### Building upon prior knowledge and indirectly related data {.page_break_before}
 
-One strategy to overcome the lack of large normalized datasets in rare disease is to combine a variety of data types and explore rare disease data alongside other knowledge.
-By using several data modalities (such as curated pathways, genetic data, or other data types), it may be possible to gain a better understanding of rare diseases (e.g., identifying novel genotype-phenotype relationships or opportunities for drug repurposing).
-Knowledge graphs (KGs) which integrate related-but-different data types, provide a rich multimodal data source (e.g., Monarch Graph Database [@doi:10.1093/nar/gkw1128], hetionet [@doi:10.7554/elife.26726], PheKnowLator [@doi:10.1101/2020.04.30.071407], and the Global Network of Biomedical Relationships [@doi:10.1093/bioinformatics/bty114], Orphanet [@http://www.orpha.net]). 
+One strategy to overcome the paucity of data in rare disease is to combine a variety of data types and explore rare disease data alongside other existing knowledge.
+By using several data modalities, such as curated pathways, genetic data, or other data types, it may be possible to gain a better understanding of rare diseases.
```

Comment:
```suggestion
By using several data modalities, such as curated pathways, or genetic data, it may be possible to gain a better understanding of rare diseases.
```